### PR TITLE
Setup script

### LINKED ISSUE #47: /configure public accessible
[/configure](http://baptiste.gelez.xyz/configure) is currently publicly accessible. This could be a security issue.

### PR DESCRIPTION
**Pull request by BaptisteGelez**
When we run Plume, check that the database is ready, and that the instance have been created. If it is not, create the database, run migrations, and ask a few questions to set up the instance and create the admin.

Fix #47

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,6 @@
 
-BASE_URL=plu.me
\ No newline at end of file
+BASE_URL=plu.me
+ROCKET_SECRET_KEY=7IdEGAf7EL7ehPjLNncCcE+2B0MTiywxDzr8KklU/VA=
```
Comment (fdb-hiroshima) [inline .env]: did you actually wanted to commit a secret key?

File: src/setup.rs [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+                .arg("plume")
+                .status()
+                .map(|s| {
+                    if s.success() {
+                        println!("{}\n", "  ✔️ Done".green());
+                    }
+                })
+                .expect("Couldn't create new user");
+
+            println!("{}\n", "About to create a new PostgreSQL table named 'plume'".blue());
```
Comment (fdb-hiroshima) [inline src/setup.rs]: I think it's not a table but a database

File: src/setup.rs [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+}
+
+fn complete_setup(conn: DbConn) {
+    quick_setup(conn);
+
+    println!("\nOn which port should Plume listen? (default: 7878)");
+    let port = read_line_or("7878");
+    write_to_dotenv("ROCKET_PORT", port);
+
+    println!("\nOn which port should Plume listen? (default: 0.0.0.0)");
```
Comment (fdb-hiroshima) [inline src/setup.rs]: Shouldn't it be on which ip?

Comment (BaptisteGelez) [reply src/setup.rs]: (oops, forgot to change that after having copy/pasted the previous lines)

**Pull request merged**